packages/vue/src/components/SpAssets/SpAssets.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <section>
+    <header class='assets-header'>
+      <h2 class='title'>Assets</h2>
+    </header>
+    <table class='assets-table'>
```

Comment:
I'm not sure table is the happiest path. Are we okay with using table instead of flex?

Reply:
we'll need to refactor this later on...